code/modules/clothing/under/costume.dm

```diff
@@ -132,6 +132,12 @@
 	icon_state = "villain"
 	can_adjust = FALSE
 
+/obj/item/clothing/under/costume/boatbomber
+	name = "gnome costume"
+	desc = "Chompski? Is that you?"
```

Comment:
you are legally obligated to add a doofenschmirtz reference to at least one of the gnome clothing items